reflex/__init__.py

```diff
@@ -20,7 +20,10 @@
     "foreach",
     "html",
     "match",
-    # "color_mode_cond",
+    "color_mode_cond",
```

Comment:
Should we namespace these as `color_mode.cond`, `color_mode.switch`, etc. ? Also, maybe we need to standardize on our terminology, since we call it "color_mode" here, but in the radix theme it's called "appearance". I'm in favor of aliasing the radix theme one to match this one.